diff -Naur gcc-3.3.2-orig/libiberty/Makefile.in gcc-3.3.2/libiberty/Makefile.in
--- gcc-3.3.2-orig/libiberty/Makefile.in	2003-03-23 11:16:03.000000000 +0100
+++ gcc-3.3.2/libiberty/Makefile.in	2003-12-25 16:51:11.000000000 +0100
@@ -260,7 +260,7 @@
 @MAINT@	echo stamp > stamp-functions
 
 INSTALL_DEST = @INSTALL_DEST@
-install: install_to_$(INSTALL_DEST) install-subdir
+install: install-subdir
 
 install_to_libdir: all
 	${mkinstalldirs} $(DESTDIR)$(libdir)$(MULTISUBDIR)
