diff -Naur nmap-3.48-orig/nbase/configure nmap-3.48/nbase/configure
--- nmap-3.48-orig/nbase/configure	2003-10-03 10:33:24.000000000 +0200
+++ nmap-3.48/nbase/configure	2003-11-13 22:37:23.000000000 +0100
@@ -5908,7 +5908,7 @@
     ;;
   *)
     specialssldir="$with_openssl"
-    CXXFLAGS="-I$with_openssl/include $CXXFLAGS"
+    CFLAGS="-I$with_openssl/include $CFLAGS"
     ;;
   esac
 
diff -Naur nmap-3.48-orig/nbase/configure.ac nmap-3.48/nbase/configure.ac
--- nmap-3.48-orig/nbase/configure.ac	2003-10-03 10:32:15.000000000 +0200
+++ nmap-3.48/nbase/configure.ac	2003-11-13 22:37:50.000000000 +0100
@@ -212,7 +212,7 @@
     ;;
   *)
     specialssldir="$with_openssl"
-    CXXFLAGS="-I$with_openssl/include $CXXFLAGS"
+    CFLAGS="-I$with_openssl/include $CFLAGS"
     ;;
   esac]
 )
diff -Naur nmap-3.48-orig/nsock/src/configure nmap-3.48/nsock/src/configure
--- nmap-3.48-orig/nsock/src/configure	2003-10-03 10:26:00.000000000 +0200
+++ nmap-3.48/nsock/src/configure	2003-11-13 23:07:49.000000000 +0100
@@ -1287,7 +1287,6 @@
      fi
    if test -d /usr/local/include; then
      CFLAGS="$CFLAGS -I/usr/local/include"
-     CXXFLAGS="$CFLAGS -I/usr/local/include"
    fi
 fi
 
@@ -3207,7 +3206,7 @@
     ;;
   *)
     specialssldir="$with_openssl"
-    CXXFLAGS="-I$with_openssl/include $CXXFLAGS"
+    CFLAGS="-I$with_openssl/include $CFLAGS"
     ;;
   esac
 
diff -Naur nmap-3.48-orig/nsock/src/configure.ac nmap-3.48/nsock/src/configure.ac
--- nmap-3.48-orig/nsock/src/configure.ac	2003-10-03 10:24:49.000000000 +0200
+++ nmap-3.48/nsock/src/configure.ac	2003-11-13 23:08:57.000000000 +0100
@@ -23,7 +23,6 @@
      fi
    if test -d /usr/local/include; then
      CFLAGS="$CFLAGS -I/usr/local/include"
-     CXXFLAGS="$CFLAGS -I/usr/local/include"
    fi
 fi
 
@@ -94,7 +93,7 @@
     ;;
   *)
     specialssldir="$with_openssl"
-    CXXFLAGS="-I$with_openssl/include $CXXFLAGS"
+    CFLAGS="-I$with_openssl/include $CFLAGS"
     ;;
   esac]
 )
