--- linux-2.4.21/mm/mremap.c.orig	2004-01-18 02:45:22.000000000 +0100
+++ linux-2.4.21/mm/mremap.c	2004-01-18 02:47:35.000000000 +0100
@@ -237,6 +237,14 @@
 		if (new_len > TASK_SIZE || new_addr > TASK_SIZE - new_len)
 			goto out;
 
+                /* the single if statement below resolves the mremap vulnerability
+                 * Allow new_len == 0 only if new_addr == addr
+                 * to preserve truncation in place (that was working
+                 * safe and some app may depend on it).
+                 */
+		if (unlikely(!new_len && new_addr != addr))
+		    goto out;
+
 		/* Check if the location we're moving into overlaps the
 		 * old location at all, and fail if it does.
 		 */
